.github/ISSUE_TEMPLATE/NOTABLE-CONTRIBUTION.md

```diff
@@ -4,7 +4,9 @@ about: Endorse yourself or someone else for a notable contribution to the projec
 ---
 
 ## Description
-Thank you for your interest in contributing to the gno.land project and joining the [Game of Realms](https://github.com/gnolang/game-of-realms/)! This template will guide you in documenting your contributions to our ecosystem. Fill out as much
+Thank you for your interest in contributing to the gno.land project and joining the [Game of Realms](https://github.com/gnolang/game-of-realms/)! This template will guide you in documenting your contributions to our ecosystem. Fill out in as many details as possible. 
```

Comment:
See my comments on #22. I believe we shouldn't use an issue template. Feel free to add useful text to the repository's top-level README instead.